2.3-tapscript.ipynb

```diff
@@ -432,7 +431,7 @@
     "print(\"pubkey0: {}\".format(pubkey0.get_bytes().hex()))\n",
     "print(\"pubkey1: {}\\n\".format(pubkey1.get_bytes().hex()))\n",
     "\n",
-    "# Method: 32B preimage - hashlib.new('sha256', bytes).digest()\n",
+    "# Method: 32B preimage - sha256('sha256', bytes)\n",
     "# Method: 20B digest - hashlib.new('ripemd160', bytes).digest()\n",
```

Comment:
Still need hashlib for this ripemd160 call

Reply:
fixed